sdk/core/azure-core-test/src/main/java/com/azure/core/test/http/TestProxyPlaybackClient.java

```diff
@@ -114,6 +120,7 @@ public Queue<String> startPlayback(File recordFile, Path testClassPath) {
                 String value = stringStringEntry.getValue();
                 strings.add(value);
             }
+            strings.addLast(xRecordingFileLocation);
```

Comment:
I would rather we change the return type here to include the recording path or make it a property on the playback client. Including a magic non-variable with the variables seems likely to cause a bug in the future.

Reply:
I thought about this. But adding it as a variable on the playback client we cannot retrieve it. We need it to be sent back from the playback client to the interceptor manager so that it can be exposed from there.

Reply:
I think that's fine.